.github/scripts/nvidia_models.py

```
@@ -25,4 +25,4 @@
 NVIDIA_MODEL_IDS: list[str] = [model_id for model_id, _label in NVIDIA_CANDIDATE_MODELS]
 
 # Keep old name as alias for backward compatibility
-CANDIDATE_MODELS = NVIDIA_CANDIDATE_MO
\ No newline at end of file
+CANDIDATE_MODELS = NVIDIA_CANDIDATE_MO
```

Ruff (0.15.21)

[error] Undefined name `NVIDIA_CANDIDATE_MO`

(F821)